config/tool_destinations.yml.sample

```diff
@@ -0,0 +1,302 @@
+# further discussed below.
+#
+# Starting with rule_type, there are currently 3 rule types: file_size, records,
+# and arguments.
+#
+# file_size and records rules are based on how large the files are: if they fall
+# within specified limits, then the rule is satisfied, and the tool may proceed
+# with the appropriate destination.
+#
+# file_size and records rules have the following required parameters on top of the base
+# mandatory parameters:
+# upper_bound
+# lower_bound
+#
+# Bounds are allowed to be specified in bytes (48000 for example) or a higher size unit,
+# including the unit abbreviation (4 GB or 10 TB for example). Additionally, upper_bound
+# is allowed to be Infinite; simply specify Infinite in order to do so.
+#
+# **The rule will allow the lower_bound, up to but not including the upper_bound
+#
+# The third rule_type is arguments, which has arguments as a mandatory parameter ontop of
+# the base mandatory parameters. The arguments parameter is specified using the following
+# template:
+#
+#
+# arguments:
+#   argument_name: the_argument
+#
+#
+# A real world example is shown below:
+#
+#
+# tools:
+#   spades:
+#     rules:
+#       - rule_type: arguments
+#         nice_value: -19
+#         destination: fail
+#         fail_message: Don't do that
+#         arguments:
+#           careful: true
+#     default_destination: cluster_low
+# default_destination: cluster
+# verbose: False
+#
+#
+# Next up, nice_value is used for prioritizing rules over others in case two rules
+# match. nice_value basically translates to, "the higher the nice_value, the 'nicer'
+# the tool is about being picked last". So based off of that idea, a rule with a nice
+# value of -5 is guaranteed to be picked over a rule with a nice value of 10. nice_value
+# is allowed to go from -20 to 20. If two rules have the same nice value and both were
+# satisfied, the first rule in the config file will be picked. In summary, first-come-
+# first-serve basis unless nice_value overrides that.
+#
+#
+# Finally, destination simply refers to the specific way the tool will run. Each
+# destination ID refers to a specific configuration to run the tool with.
+#
+# Some rules may call for the job to fail if certain conditions are encountered. In
+# this case, destination simply refers to 'fail'.
+#
+# For example, the following rule is set to fail the job if a file that is too large
+# (more than 4GB) is encountered:
+#
+#
+# tools:
+#   spades:
+#     rules:
+#       - rule_type: file_size
+#         nice_value: 0
+#         destination: fail
+#         fail_message: Data too large
+#         lower_bound: 4 GB
+#         upper_bound: Infinity
+#
+#
+# As shown above, a rule with 'fail' as the destination requires an additional
+# parameter, 'fail_message', which DynamicToolDestination uses to print a helpful error
+# message to the user indicating why the job failed (showing up inside the job log in
+# Galaxy's history panel).
+
+
+tools:
```

Comment:
Do you mean should the whole file be commented out? If so yes, is that what is done with out sample config files?